cellprofiler/gui/imagesetctrl.py

```diff
@@ -202,7 +203,7 @@ def ordering_fn(a, b):
                 COL_ORDER.index(a.column_type), COL_ORDER.index(b.column_type)
             )
 
-        columns.sort(cmp=ordering_fn)
+        columns = sorted(columns, key=functools.cmp_to_key(ordering_fn))
```

Comment:
Maybe outside the scope of this PR, but should `ordering_fn` be rewritten so that `cmp_to_key` is no longer needed?

Reply:
I think that's a good idea considering python 3.7+ doesn't appear to support the `cmp` argument anymore. i'm thinking it'll be easier if i make the change in this pr.

Reply:
Sounds good, feel free to ping me when you have that ready

Reply:
hey, so, after looking at the source, I'm starting to think it'll be better if we continue to use `cmp_to_key()` since it's appears to be in a "refactored" form already. What do you think? (Source Snippet: http://code.activestate.com/recipes/576653-convert-a-cmp-function-to-a-key-function/)

Reply:
@AetherUnbound and @jeffsyang Yeah. You want to use `sorted` with the `key` parameter. This works.